scripts/kind-e2e/e2e.sh

```diff
@@ -70,8 +70,7 @@ function setup_custom_cni(){
             echo "Applying weave network in to cluster${i}..."
             kubectl --context=cluster${i} apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')&env.IPALLOC_RANGE=${cluster_CIDRs[cluster${i}]}"
             echo "Waiting for weave-net pods to be ready cluster${i}..."
-            # FIXME: This timeout doesn't need to be so long
-            kubectl --context=cluster${i} wait --for=condition=Ready pods -l name=weave-net -n kube-system --timeout=700s
+            kubectl --context=cluster${i} wait --for=condition=Ready pods -l name=weave-net -n kube-system --timeout=300s
```

Comment:
I'm fine with lowering it but it should be uncommon for it to timeout so having it longer to give more safety cushion shouldn't hurt.

Reply:
The problem is when it correctly-fails and you have to wait 700s to find it out.

Reply:
yeah but I'd think that **should never** happen? Either way I'm fine to lower it as long as we're sure 5 min gives it plenty of cushion.

Reply:
It has happened to me quite a bit as I've been developing the e2e scripts. If everything is working correctly, it shouldn't be hit, but as it's being developed or if something is wrong, it is sometimes hit.